docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
Changing array to iterable is fine with me, as long as we do it now, before a new release. 

But do we have a definition of iterable? Objects are iterable in a sense too (`for ... in`) ... Is it anything that has a length property?